### ISSUE TITLE
Should we change the default blocking io dispatcher size?

### ISSUE THREAD

**Issue by He-Pin** [apache/incubator-pekko#1213]
Motivation:
The current default blocking io dispatcher is fixed size 16, I think it's too big.
![image](https://github.com/apache/incubator-pekko/assets/501740/4567619b-780e-4724-b3cb-0eced7af3fa1)

Better with a non-fixed-sized one?eg default to 1, and size up to 16 if needed. But yes, user can adjust this themself.

**Comment by laglangyue** [comment]
I agree that it can be changed. And default value should be 16, because users can upgrade without being affected.
